lib/sles4sap_publiccloud.pm

```diff
@@ -608,4 +610,48 @@ sub delete_network_peering {
     }
 }
 
+=head2 get_hana_database_info
+
+    Parses "hdbsql" command output.
+    Returns true if hana database is online.
```

Comment:
my 2 cents, describe the return in term of 0/1 and not in term of true/false
please mind also to explain what `-1` stand for

Reply:
refactored